common/changes/@uifabric/example-app-base/FacepileResizeGroup_2017-10-03-18-34.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
```

Comment:
You didn't touch any package except for office-ui-fabric-react, there shouldn't be any changefiles for the other packages